subkey/src/main.rs

```diff
@@ -14,6 +14,16 @@ fn good_waypoint(done: u64) -> u64 {
 	}
 }
 
+fn next_seed(mut seed: [u8; 32]) -> [u8; 32] {
+	for i in 0..32 {
+		match seed[i] {
+			255 => { seed[i] = 0; }
```

Comment:
after 255 iterations this will be exactly the same, won't it? are we _sure_ this has enough entropy for actually generating keys correctly?

Reply:
that byte will be the same, sure, but the next will increment. this gets fed into a SHA2-512 hash as part of the Ed25519 keygen, so, yeah i'd say it'll be fine.